tests/acceptance/bootstrap/SearchContext.php

```diff
@@ -220,4 +245,31 @@ public function theSearchResultShouldContainEntriesWithHighlight(
 			);
 		}
 	}
+
+	/**
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder|space) "([^"]*)" using the WebDAV API$/
+	 * @When /^user "([^"]*)" searches for "([^"]*)" inside (folder) "([^"]*)" in space "([^"]*)" using the WebDAV API$/
```

Comment:
```suggestion
	 * @When /^user "([^"]*)" searches for "([^"]*)" inside folder "([^"]*)" in space "([^"]*)" using the WebDAV API$/
```
I don't think it is required

Reply:
IMHO, `()` works as a placeholder here. So I think it is required. removing that caused some tests to fail.

Reply:
If so that's fine. Actually, I checked at some places and didn't see it was used as a placeholder.

Reply:
If so that's fine